devices.js

```diff
@@ -4348,6 +4348,21 @@ const devices = [
         fromZigbee: [fz.iaszone_occupancy_1],
         toZigbee: [],
     },
+    {
+        zigbeeModel: ['PIRSensor-EM'],
+        model: 'HS1MS',
+        vendor: 'HEIMAN',
+        description: 'Smart motion Sensor',
+        supports: 'occupancy',
+        fromZigbee: [fz.iaszone_occupancy_1],
```

Comment:
Battery reporting is configured below but there is no corresponding converter, is `battery_3V` missing from here?

Reply:
I add battery_3V not found any other attribute report